### DISCUSSION TITLE
Bumping from alpha to another alpha with different minor

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [renovatebot/renovate#16087] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I have a dependency that was using the version 0.1.0-alpha.10, after a new release to 0.2.0-alpha.1, Renovate doesn't create a PR for it.

public repo: https://github.com/javiersc/gradle-plugins/

### Relevant debug logs

<details><summary>Logs</summary>

> There was an error creating your Issue: body is too long (maximum is 65536 characters).

As I am unable to create the issue with the logs, I have pasted them in a pastebin.

https://pastebin.com/V7ttK9bQ
</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#ignoreunstable

**Reply by JavierSegoviaCordoba** [reply]
Not what I want I think, I want to update from unstable to another unstable with different scope, but when an unstable becomes eventually final, I don't want to get more unstable updates for that final.

If I set `ignoreUnstable` to `true`, I will get unstable updates for final versions.

**Reply by rarkins** [reply]
Definitely not a bug though, so moving this to a discussion is appropriate.

You want to upgrade from current unstable to new unstable only if there exists no stable release in-between? e.g. `0.2.0`?

**Reply by JavierSegoviaCordoba** [reply]
Exactly

**Reply by rarkins** [reply]
Not something we can do today and sounds like quite an edge case. Does this package use a tagging like `next` you can use instead with our option `followTag`?
